force-app/main/default/lwc/donationHistoryTable/donationHistoryTable.js

```diff
@@ -33,17 +33,30 @@ export default class DonationHistoryTable extends LightningElement {
         if (data) {
             this.paymentMethodLabel = data.fields[PAYMENT_FIELD.fieldApiName].label
         };
-        this.columns = [
-            { label: RD2_ScheduleVisualizerColumnDate, fieldName: 'closeDate', type: 'date-local', typeAttributes:{
-                year: "numeric",
-                month: "numeric",
-                day: "numeric",
-            },
-            cellAttributes: { alignment: 'right' }},
-            { label: donationHistoryDonorLabel, fieldName: 'name', type: 'text' },
-            { label: commonAmount, fieldName: 'amount', type: 'currency', },
-            { label: this.paymentMethodLabel, fieldName: 'paymentMethod', type: 'text', },
-        ];
+        arePaymentsEnabled({contactId: this.contactId}).then( (paymentsAreEnabled) => {
```

Comment:
We can probably remove the whole `@wire(getObjectInfo, { objectApiName: DATA_IMPORT })` method by making one further change to `getDonationHistory()`. Lets include the payment method label in the new DonationHistoryAppView class.